src/components/NewMovie/NewMovie.js

```diff
@@ -1,12 +1,135 @@
 import React, { Component } from 'react';
+import './NewMovie.scss';
+import classNames from 'classnames';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    const newMovie = {
+      title: this.state.title,
+      description: this.state.description,
+      imgUrl: this.state.imgUrl,
+      imdbUrl: this.state.imdbUrl,
+      imdbId: this.state.imdbId,
+    };
+
+    const { title, description, imgUrl, imdbUrl, imdbId } = this.state;
```

Comment:
desctructure state before creating newMovie object, so you won't need to write this.state etc
```suggestion
    const { title, description, imgUrl, imdbUrl, imdbId } = this.state;
    
    const newMovie = {
      title,
      description,
      imgUrl,
      imdbUrl,
      imdbId,
    };
```